js/pillbox.js

```diff
@@ -348,19 +353,20 @@
 
 		inputEvent: function inputEvent (e) {
 			var self = this;
-			var cleanInput = this.options.cleanInput;
```

Comment:
in my opinion scoping the `cleanInput` method locally is a better option. less object dot chains. another nice perk it makes for better uglifying.